modules/util/portal-tools-rest-builder/src/main/resources/com/liferay/portal/tools/rest/builder/dependencies/base_resource_impl.ftl

```diff
@@ -309,7 +309,7 @@ public abstract class Base${schemaName}ResourceImpl
 		</#if>
 		@Override
 		${freeMarkerTool.getResourceMethodAnnotations(javaMethodSignature)}
-		public <#if generatePermissions>final</#if> ${javaMethodSignature.returnType} ${javaMethodSignature.methodName}(${freeMarkerTool.getResourceParameters(configYAML, javaMethodSignature.javaMethodParameters, javaMethodSignature.operation, allSchemas, true)}) throws Exception {
+		public <#if generatePermissions>final</#if> ${javaMethodSignature.returnType} <#if javaMethodSignature.requestBodyMediaTypes?? && javaMethodSignature.requestBodyMediaTypes?seq_contains("multipart/form-data")>${javaMethodSignature.methodName}Multipart<#else>${javaMethodSignature.methodName}</#if>(${freeMarkerTool.getResourceParameters(configYAML, javaMethodSignature.javaMethodParameters, javaMethodSignature.operation, allSchemas, true)}) throws Exception {
```

Comment:
I don't think this is the best way to proceed. Why don't we update the method name somewhere? With that, we will have the method name everywhere updated; otherwise we need to update the method name here, also in the GraphQL template, etc.

Otherwise, maintaining this could become a nightmare